### PR TITLE
refactor(rendering 'your name' for 'appellant name' if not agent): re…

### PR DESCRIPTION
**Pull request by biroldif**
…nder 'Your name' if not agent

## Ticket Number
https://pins-ds.atlassian.net/browse/AS-2307

## Description of change
rendering 'your name' for 'appellant name' if not agent

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/forms-web-app/src/views/your-planning-appeal/index.njk [hunk 1]
```diff
@@ -13,6 +13,17 @@
   {{ title }}
 {% endblock %}
 
+      {%- if item.status != 'CANNOT START YET' %}
```
Comment (swdpcomputing) [inline packages/forms-web-app/src/views/your-planning-appeal/index.njk:16]: Is this block indented correctly?

**Review by swdpcomputing — CHANGES_REQUESTED** [review]

**Comment by swdpcomputing** [comment]
Was this PR posted in build teams?

**Comment by karunak21** [comment]
> Was this PR posted in build teams?
It was not posted in build teams, this is a change to the Cypress e2e tests. Shall I ask Birol to post it?

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Review by karunak21 — APPROVED** [review]
All the cypress tests looks good to me

**Pull request merged**